client/src/components/SliderCard/SliderCard.module.css

```diff
@@ -20,8 +20,7 @@
 .sliderCardImg {
   position: relative;
   top: -10px;
-  width: 40%;
-  min-height: 68px;
+  width: 30%;
```

Comment:
That I changed because we had a small scroll on iphone 4 because all images was a bit different sizes